dashboard/src/views/Home.vue

```diff
@@ -89,73 +82,85 @@
           </thead>
           <tbody>
             <tr v-for="(value, key) in sensorInfo" :key="key">
-              <td scope="row">{{value.date}}</td>
+              <td scope="row">{{new Date(value.date).toLocaleString('en-GB')}}</td>
               <td>{{value.people.people}}</td>
               <td>
-                {{value.co2[0].co2}}
+                {{value.co2.length > 0 ? value.co2[0].co2: "-"}}
                 <br />
-                ({{value.co2[0].device}})
+                ({{value.co2.length > 0 ? value.co2[0].device: "-"}})
               </td>
+
               <td>
-                {{value.multi[0].hum}}
+                {{value.multi.length > 0 ? value.multi[0].hum: "-"}}
                 <br />
-                ({{value.multi[0].device}})
+                ({{value.multi.length > 0 ? value.multi[0].device: "-"}})
               </td>
               <td>
-                {{value.multi[1].hum}}
+                {{value.multi.length > 0 ? value.multi[1].hum: "-"}}
```

Comment:
`value.multi.length > 0` doesn't imply that `value.multi[1]` is not undefined. This could lead to an unexpected error when the data in `value.multi` are missing.

Reply:
fixed!